ddp-workspace/projects/ddp-sdk/src/lib/components/age-up/verifyAgeUp.component.ts

```diff
@@ -24,9 +24,7 @@ export class VerifyAgeUpComponent implements OnInit {
     }
 
     private verifyInvitation(invitationId: string): void {
-        this.invitationsService.verify(invitationId).pipe(
-            take(1)
-        ).subscribe({
+        this.invitationsService.verify(invitationId).subscribe({
             complete: () => this.nextUrl.emit(this.url),
```

Comment:
You took out the `take(1)`. Not worried about disposing of subscription?

Reply:
Yes, because this observable has only two states: complete and error, it means that observable finished, so we don't need to unsubscribe.

Reply:
Being reading (more!!) about the unsubscribe issue.
I am not saying you are wrong, but I have encountered documentation or StackOverflow entry that would indicate that the subscription would be cleaned up using this type of configuration.
Can you point to a reference? If not, then adding the `take(1)` is not a big deal anyways.

Reply:
https://stackoverflow.com/a/45478765